coral-schema/src/test/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverterTests.java

```diff
@@ -650,6 +650,19 @@ public void testCompatibleUnion() {
     Assert.assertEquals(actualSchema.toString(true), TestUtils.loadSchema("testCompatibleUnion-expected.avsc"));
   }
 
+  @Test
+  public void testUnionForceLowercase() {
+    String viewSql = "CREATE VIEW v AS " + "SELECT * FROM basecomplex " + "UNION ALL "
+        + "SELECT * FROM basecomplexunionforcelowercase";
+
+    TestUtils.executeCreateViewQuery("default", "v", viewSql);
+
+    ViewToAvroSchemaConverter viewToAvroSchemaConverter = ViewToAvroSchemaConverter.create(hiveMetastoreClient);
+    Schema actualSchema = viewToAvroSchemaConverter.toAvroSchema("default", "v", false, true);
+
+    Assert.assertEquals(actualSchema.toString(true), TestUtils.loadSchema("testUnionForceLowercase.avsc"));
```

Comment:
Create another result file `testUnionForceLowercase-expected.avsc` to be aligned with other existing tests?

Reply:
Thanks for catching this. Modified.